arangodb-net-standard/Serialization/ApiClientSerializationOptions.cs

```diff
@@ -22,20 +22,39 @@ public class ApiClientSerializationOptions
         /// </summary>
         public bool UseStringEnumConversion { get; set; }
 
+        /// <summary>
+        /// When enabled will never use camel case property names, 
+        /// will always include null values, for dictionary types only. 
+        /// This applies only to types assignable to <see cref="System.Collections.IDictionary" />.
+        /// </summary>
+        public bool UseSpecialDictionaryHandling { get; set; }
+
         /// <summary>
         /// Create serialization options.
         /// </summary>
-        /// <param name="useCamelCasePropertyNames">Whether property names should be serialized using camelCase.</param>
-        /// <param name="ignoreNullValues">Whether null values should be ignored - i.e. not defined at all in the serialized string.</param>
-        /// <param name="useStringEnumConversion">Whether to serialize enum values to a string value instead of an integer.</param>
+        /// <param name="useCamelCasePropertyNames">
+        /// Whether property names should be serialized using camelCase.
+        /// </param>
+        /// <param name="ignoreNullValues">
+        /// Whether null values should be ignored - i.e. not defined at all in the serialized string.
+        /// </param>
+        /// <param name="useStringEnumConversion">
+        /// Whether to serialize enum values to a string value instead of an integer.
+        /// </param>
+        /// <param name="useSpecialDictionaryHandling">
+        /// Whether to use the special dictionary handling,
+        /// which overrides useCamelCasePropertyNames and ignoreNullValues for dictionary types only.
```

Comment:
Maybe we should indicate explicitly what the behavior will be?

> overrides useCamelCasePropertyNames and ignoreNullValues"

As a user of the API, I would ask myself: Will they use camel case and ignore null values or not? To find out, I will need to look at the `DictionaryValueConverter` class, which isn't obvious.

Reply:
OK I can re-use similar wording I've added on the property itself